Messenger/AppDelegate.mm

```diff
@@ -172,11 +172,16 @@ - (void)updateWindowTitlebar {
   updateButton([_window standardWindowButton:NSWindowZoomButton]);
 }
 
+- (NSInteger) activeConversationIndex {
+    NSString* script = @"sel=document.querySelector('li._1ht2');for(var i=0;i<sel.parentNode.childNodes.length;i++){var item=sel.parentNode.childNodes[i];if(item==sel){activeConversationIndex=i}}";
```

Comment:
This is shaky and very unreliable. You definitely can *not* rely on css classnames and *probably not* even on DOM ids.

Further, the better solution here would be relative moves instead of absolute — nextSibling, previousSibling is a better approach.